projects/prometheus/prometheus.yml

```diff
@@ -0,0 +1,165 @@
+rule_files:
```

Comment:
How are we going to keep this up to date every time we add something to origin?

Reply:
If this file has a canonical location, I think we can use that. We need our own `prometheus.rules` and `alertmanager.yml`.